src/status_im/ui/screens/status/new/views.cljs

```diff
@@ -53,7 +54,8 @@
         input-text (re-frame/subscribe [:chats/timeline-chat-input-text])
         sending-image (re-frame/subscribe [:chats/timeline-sending-image])]
     (fn []
-      (let [{:keys [uri]} (first (vals @sending-image))]
+      (let [{:keys [uri]} (first (vals @sending-image))
+            text-lenght (count @input-text)]
```

Comment:
text-length and message-max-length are misspelled

Reply:
oh jeez... I will fix it, thanks